### ISSUE TITLE
PHP_CodeCoverage_Exception from running phpcov

### ISSUE THREAD

**Issue by bskendig** [sebastianbergmann/phpcov#35]
When I run phpcov on a file that was generated by php-code-coverage(latest version, 2.0.15, following the "Using" section in its README to generate the file with PHP_CodeCoverage_Report_PHP), I get an exception.

```
tmp $ head code-coverage-report 
<?php
$coverage = new PHP_CodeCoverage;
$coverage->setData(array (
  '/p4-wdpro/Projects/NGE/Sites/pep/main/target/packaged-application/library/Zend/Controller/Plugin/Abstract.php' => 
  array (
    50 => 
    array (
    ),
    51 => 
    array (
tmp $ phpcov -vvv --html=foo.html execute code-coverage-report 
phpcov 2.0.1 by Sebastian Bergmann.



                                
  [PHP_CodeCoverage_Exception]  
                                


Exception trace:
 () at phar:///usr/local/bin/phpcov/phpunit/php-code-coverage/src/CodeCoverage.php:319
 PHP_CodeCoverage->append() at phar:///usr/local/bin/phpcov/phpunit/php-code-coverage/src/CodeCoverage.php:295
 PHP_CodeCoverage->stop() at phar:///usr/local/bin/phpcov/src/ExecuteCommand.php:154
 SebastianBergmann\PHPCOV\ExecuteCommand->execute() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Command/Command.php:252
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:887
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:193
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/phpcov/src/Application.php:98
 SebastianBergmann\PHPCOV\Application->doRun() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:124
 Symfony\Component\Console\Application->run() at /usr/local/bin/phpcov:552


execute [--configuration="..."] [--blacklist="..."] [--whitelist="..."] [--add-uncovered] [--process-uncovered] [--clover="..."] [--crap4j="..."] [--html="..."] [--php="..."] [--text="..."] script
```

I get the same results if I omit "--html=foo.html".

**Comment by bskendig** [comment]
Perhaps I'm not running it correctly, because "phpcov help execute" says its argument is "script", and the coverage file isn't a "script"... so I tried using merge instead:

```
$ mv code-coverage-report code-coverage-report.cov
$ phpcov merge /tmp --html=foo -vvv
phpcov 2.0.1 by Sebastian Bergmann.


Generating code coverage report in HTML format ...

                                      
  [InvalidArgumentException]          
  Template file could not be loaded.  
                                      


Exception trace:
 () at phar:///usr/local/bin/phpcov/phpunit/php-text-template/Text/Template.php:112
 Text_Template->setFile() at phar:///usr/local/bin/phpcov/phpunit/php-text-template/Text/Template.php:88
 Text_Template->__construct() at phar:///usr/local/bin/phpcov/phpunit/php-code-coverage/src/CodeCoverage/Report/HTML/Renderer/Directory.php:65
 PHP_CodeCoverage_Report_HTML_Renderer_Directory->render() at phar:///usr/local/bin/phpcov/phpunit/php-code-coverage/src/CodeCoverage/Report/HTML.php:143
 PHP_CodeCoverage_Report_HTML->process() at phar:///usr/local/bin/phpcov/src/BaseCommand.php:216
 SebastianBergmann\PHPCOV\BaseCommand->handleReports() at phar:///usr/local/bin/phpcov/src/MergeCommand.php:129
 SebastianBergmann\PHPCOV\MergeCommand->execute() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Command/Command.php:252
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:887
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:193
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/phpcov/src/Application.php:98
 SebastianBergmann\PHPCOV\Application->doRun() at phar:///usr/local/bin/phpcov/symfony/console/Symfony/Component/Console/Application.php:124
 Symfony\Component\Console\Application->run() at /usr/local/bin/phpcov:552


merge [--clover="..."] [--crap4j="..."] [--html="..."] [--php="..."] [--text="..."] directory
```

When I look at line 143 of https://github.com/sebastianbergmann/php-code-coverage/blob/master/src/CodeCoverage/Report/HTML.php, that's the part where it's trying to copy some css files (css/bootstrap.min.css, css/nv.d3.css, css/style.css), but I don't see these files in the phar or anywhere else on my system. Maybe this error is because the files are missing?

**Comment by bskendig** [comment]
In my first comment above, I misunderstood the readme and I was running phpcov incorrectly.

In my second comment above, I was hitting https://github.com/sebastianbergmann/phpcov/issues/19. I was able to work around it by using the version of phpcov that I got from Composer instead of the phar that I downloaded. It looks like the phar is missing some files.

The phpcov documentation could be better; for future reference, after I generated one or more .cov files in /tmp, here's the command I used to generate an html report:

    /path/to/vendor/bin/phpcov merge --html=myhtmldir /tmp

This is no longer an issue; closing.
